check_correlated_env.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
```

Comment:
Is it normal that we have exactly the same execution when selecting all the same actions? (cf check_env_correlated)
Maybe we need to think about randomization when sampling from the npz file.

Reply:
If the actions are 'do-nothing-no-inspection' or 'repair', both environments should behave similarly.
The correlation formation will be updated only when info from inspections arrives.

Reply:
What I meant is that I instantiated 2 environments and played the same (but random) actions in both, resulting in the same sum of rewards over several episodes.